lms/static/sass/shared/_footer.scss

```diff
@@ -358,9 +374,14 @@ $edx-footer-bg-color: rgb(252,252,252);
       display: inline-block;
       margin-bottom: ($edx-footer-spacing/2);
     }
+
+    @media print {
```

Comment:
in my testing here, adding float: left to this area ensured that the print footer copyright text and the openedx logo is presented in the same line instead of being stacked. 

This is what I'm seeing without the float: left rule: 
![image](https://cloud.githubusercontent.com/assets/2023680/7619318/db505a96-f985-11e4-89f5-101c7a791734.png)